core/math/octree_definition.inc

```diff
@@ -929,6 +931,9 @@ OCTREE_FUNC(OctreeElementID)::create(T *p_userdata, const AABB &p_aabb, int p_su
 	return last_element_id - 1;
 }
 
+OCTREE_FUNC(void)::update() {
```

Comment:
Why is this update function necessary? I only notice it used in the broad phase octree updated function which was previously empty.

Reply:
Good spot, it looks like a stub that was used during development. I'll see if it can be removed. :+1: